src/js/trove/cmdline-lib.js

```diff
@@ -5,8 +5,13 @@
   theModule: function(runtime, _, uri) {
     return runtime.makeModuleReturn(
       {
-        "command-line-arguments": runtime.makeFunction(function() {
-          return runtime.ffi.makeList(runtime.getParam("command-line-arguments").map(runtime.makeString));
+          "command-line-arguments": runtime.makeFunction(function() {
+              var args = runtime.getParam("command-line-arguments");
+              if (args.length == 0) {
+                  args = ["<browser>"];
```

Comment:
@puppyofkosh clarification q – is this saying “the executable for this run is the browser, not node”?